staging_manifest.yml

```diff
@@ -13,9 +13,10 @@ services:
 - federalist-staging-s3
 - federalist-staging-env
 env:
-  NODE_ENV: staging
+  NODE_ENV: production
+  APP_ENV: staging
   SAILS_LOG_LEVEL: verbose
-  NPM_CONFIG_PRODUCTION: false
+  NPM_CONFIG_PRODUCTION: true
```

Comment:
I did test that! Sails boots up in production mode, probably because it uses `npm run start` to execute the app.js file.